tests/display/DisplayObject.tests.ts

```diff
@@ -0,0 +1,311 @@
+import { DEG_TO_RAD, RAD_TO_DEG } from '../../src/maths/misc/const';
+import { Container } from '../../src/scene/container/Container';
+import { updateLayerGroupTransforms } from '../../src/scene/container/utils/updateLayerGroupTransforms';
+
+describe('Container', () =>
```

Comment:
Lets rename this File to something else - DisplayObject is not a thing. 
As we could end up with one uber file - perhaps we break container tests up in to smaller groups?

Reply:
Sorted.